src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
solc, the solidity compiler commandline interface
Version: 0.4.10+commit.59099908.Linux.g++

I think I know what's going on...`^` in `pragma solidity ^0.4.9;` probably means "use at least this version or newer"